lib/rubocop/cop/lint/debugger.rb

```diff
@@ -72,8 +51,23 @@ def message(node)
           format(MSG, source: node.source)
         end
 
-        def binding_irb?(node)
-          binding_irb_call?(node)
+        def with_receiver?(node)
```

Comment:
I see you're using this method only once, so I'd inline it in `forbidden_receiver?`.